src/main.cc

```diff
@@ -178,6 +178,35 @@ signal_cb(ev_loop *loop, struct ev_signal *w, int revents)
 	tarantool_exit(0);
 }
 
+static void
```

Comment:
Don't forget to rebase on top of a fresh `master` at next update.

Reply:
Done.